assets/src/customizer/components/google-fonts-control/index.js

```diff
@@ -131,6 +132,11 @@ const GoogleFontsControl = props => {
 		setItems( newChildren );
 	};
 
+	const onFail = error => {
+		console.error( error );
```

Comment:
I wonder if we should display a more visible error message, maybe we can use the [notifications API](https://make.wordpress.org/core/2016/07/05/customizer-apis-in-4-6-for-setting-validation-and-notifications/)

Reply:
I _think_ I did this correctly now...